### PR TITLE
Create team.php

### LINKED ISSUE #117: Block Patterns - Team
**Description**

🛠️ The goal of this issue is to track the implementation of the "Team" pattern.
🎨 [Figma](https://www.figma.com/file/AlYr03vh4dVimwYwQkTdf6/Twenty-Twenty-Four?type=design&node-id=565%3A10409&mode=design&t=LHSPHKUrPq556l59-1)

Team

### PR DESCRIPTION
**Pull request by marcelle42**
Created team block pattern

### CODE REVIEW

File: patterns/team.php [hunk 1]
```diff
@@ -0,0 +1,34 @@
+<?php
+/**
+ * Title: Team
+ * Slug: twentytwentyfour/team
+ * Categories: team
+ * Block Types: core/template-part/team
```
Comment (MaggieCabrera) [inline patterns/team.php]: This only works if we are targeting a specific block, we can remove this line

File: patterns/team.php [hunk 2]
```diff
@@ -0,0 +1,34 @@
+<?php
+/**
+ * Title: Team
+ * Slug: twentytwentyfour/team
+ * Categories: team
```
Comment (MaggieCabrera) [inline patterns/team.php]: Lets use the default categories even if the figma says otherwise

File: patterns/team.php (hunk 1 above)
Comment (marcelle42) [reply patterns/team.php]: Fixed

File: patterns/team.php (hunk 2 above)
Comment (marcelle42) [reply patterns/team.php]: Fixed

File: patterns/project-description.php [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+<div class="wp-block-group has-custom-brown-background-color has-background" style="padding-top:75px;padding-right:101px;padding-bottom:75px;padding-left:101px"><!-- wp:columns -->
+<div class="wp-block-columns"><!-- wp:column {"style":{"spacing":{"blockGap":""}}} -->
+<div class="wp-block-column"><!-- wp:paragraph {"fontSize":"small"} -->
+<p class="has-small-font-size"><?php echo esc_html_x('Piva Hotel — Overview','Project Description pattern header','twentytwentyfour'); ?></p>
+<!-- /wp:paragraph --></div>
+<!-- /wp:column -->
+
+<!-- wp:column {"style":{"spacing":{"blockGap":""}}} -->
+<div class="wp-block-column"><!-- wp:paragraph {"fontSize":"medium"} -->
+<p class="has-medium-font-size"><?php echo esc_html_x('Sessione fotografica per il Piva Arts &amp; Crafts Hotel vicino al lago Piva, in Montenegro. Un lago artificiale è stato costruito nel 1975, dividendo il Piva Kanyon per costruire una diga Mratinje.','Project Description pattern text','twentytwentyfour');?></p>
```
Comment (luminuu) [inline patterns/project-description.php:19]: We should have only English in the copy for easier translation into other languages.

File: patterns/team.php [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+<h2 class="wp-block-heading has-text-align-center has-large-font-size"><?php echo esc_html_x('Meet our team', 'Header of team pattern','twentytwentyfour');?></h2>
+<!-- /wp:heading -->
+
+<!-- wp:paragraph {"align":"center","fontSize":"small"} -->
+<p class="has-text-align-center has-small-font-size"><?php echo esc_html_x('Our comprehensive suite of professionals caters to a diverse team, <br>ranging from seasoned architects to renowned engineers.','Subheader of team pattern','twentytwentyfour');?></p>
+<!-- /wp:paragraph -->
+
+<!-- wp:group {"style":{"spacing":{"blockGap":"3em"}},"layout":{"type":"flex","flexWrap":"wrap","justifyContent":"center"}} -->
+<div class="wp-block-group"><!-- wp:image {"align":"center","id":19,"aspectRatio":"1","scale":"cover","sizeSlug":"full","linkDestination":"none","className":"is-style-rounded"} -->
+<figure class="wp-block-image aligncenter size-full is-style-rounded"><img src="<?php esc_url(get_template_directory_uri().'assets/team-member-1.jpg');?>" alt="" class="wp-image-19" style="aspect-ratio:1;object-fit:cover"/><figcaption class="wp-element-caption"><strong>Francesca Piovani</strong><br>Founder, CEO &amp; Architect</figcaption></figure>
```
Comment (luminuu) [inline patterns/team.php:19]: Make the name and title translateable

File: patterns/team.php [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+<!-- /wp:image -->
+
+<!-- wp:image {"align":"center","id":20,"aspectRatio":"1","scale":"cover","sizeSlug":"full","linkDestination":"none","className":"is-style-rounded"} -->
+<figure class="wp-block-image aligncenter size-full is-style-rounded"><img src="<?php esc_url(get_template_directory_uri().'assets/team-member-2.jpg');?>" alt="" class="wp-image-20" style="aspect-ratio:1;object-fit:cover"/><figcaption class="wp-element-caption"><strong>Rhye Moore</strong><br>Engineering Manager</figcaption></figure>
```
Comment (luminuu) [inline patterns/team.php:23]: See above

File: patterns/team.php [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+<!-- /wp:image -->
+
+<!-- wp:image {"align":"center","id":22,"aspectRatio":"1","scale":"cover","sizeSlug":"full","linkDestination":"none","className":"is-style-rounded"} -->
+<figure class="wp-block-image aligncenter size-full is-style-rounded"><img src="<?php esc_url(get_template_directory_uri().'assets/team-member-3.jpg');?>" alt="" class="wp-image-22" style="aspect-ratio:1;object-fit:cover"/><figcaption class="wp-element-caption"><strong>Helga Steiner</strong><br>Architect</figcaption></figure>
```
Comment (luminuu) [inline patterns/team.php:27]: See above

File: patterns/team.php [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+<!-- /wp:image -->
+
+<!-- wp:image {"align":"center","id":21,"aspectRatio":"1","scale":"cover","sizeSlug":"full","linkDestination":"custom","className":"is-style-rounded"} -->
+<figure class="wp-block-image aligncenter size-full is-style-rounded"><img src="<?php esc_url(get_template_directory_uri().'assets/team-member-4.jpg');?>" alt="" class="wp-image-21" style="aspect-ratio:1;object-fit:cover"/><figcaption class="wp-element-caption"><strong>Ivan Lawrence</strong><br>Project Manager</figcaption></figure>
```
Comment (luminuu) [inline patterns/team.php:31]: See above

**Review by luminuu — CHANGES_REQUESTED** [review]
Added some feedback on this PR:

File: patterns/project-description.php (hunk 3 above)
Comment (kafleg) [reply patterns/project-description.php:19]: But the design have non-english text. What should we do in that case?

File: patterns/project-description.php [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+<!-- /wp:paragraph --></div>
+<!-- /wp:column --></div>
+<!-- /wp:columns -->
+
+<!-- wp:image {"id":26,"aspectRatio":"16/9","scale":"cover","sizeSlug":"large","linkDestination":"none","className":"is-style-default"} -->
+<figure class="wp-block-image size-large is-style-default"><img src="<?php esc_url(get_template_directory_uri().'assets/small-boat-in-big-water-1024x495.jpg';?>" alt="" class="wp-image-26" style="aspect-ratio:16/9;object-fit:cover"/></figure>
```
Comment (kafleg) [inline patterns/project-description.php]: Syntax error found, unexpected ';', expecting ')'

File: patterns/project-description.php (hunk 3 above)
Comment (carolinan) [reply patterns/project-description.php:19]: Replace it.

File: patterns/project-description.php (hunk 8 above)
Comment (carolinan) [inline patterns/project-description.php]:
```suggestion
<figure class="wp-block-image size-large is-style-default"><img src="<?php esc_url(get_template_directory_uri().'assets/small-boat-in-big-water-1024x495.jpg');?>" alt="" class="wp-image-26" style="aspect-ratio:16/9;object-fit:cover"/></figure>
```

**Pull request closed without merging**